pkg/resourceoverrides/overrides.go

```diff
@@ -70,19 +70,19 @@ type ResourceOverrides struct {
 	Overrides []ResourceOverride
 }
 
-type ResourceOverridesHandler struct {
+type ROHandler struct {
```

Comment:
Handler was not available? :-)

Reply:
Yeah there's a Handler variable here:

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/09c6b365bb1e416e555f86f27011047e449e70dc/pkg/resourceoverrides/overrides.go#L83

It was either this or changing all the references to that variables and since this struct was only used in this file I decided to go with renaming it here.